### ISSUE TITLE
Schemas?

### ISSUE THREAD

**Issue by icirellik** [balderdashy/sails-postgresql#182]
How do I create a model that uses postgres schema?

```
select * from schema1.table;
```

It looks like there was support and it was wiped out during the switchover to waterline-sequel.

**Comment by snovak7** [comment]
Yes, I am having the same problem...
https://gist.github.com/snovak7/5df976aba0681bd224ed

**Comment by sailsbot** [comment]
Thanks for posting, @icirellik.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
